configure/backups.md

```diff
@@ -2,31 +2,62 @@ Providing Backups
+Backups schedule is defined in the  ``backup`` section of the [deploy/cr.yaml](https://github.com/Percona-Lab/percona-server-mongodb-operator/blob/master/deploy/cr.yaml) file. 
 This section contains three subsections:
-* `s3` subsection contains data needed to access the Amazon S3 cloud to store backups.
-* `coordinator` subsections allows to configure Kubernetes limits and claims for the Percona Backup for MongoDB Coordinator daemon.
+* `storages` subsection contains data needed to access the S3-compatible cloud to store backups.
+* `coordinator` subsection allows to configure Kubernetes limits and claims for the Percona Backup for MongoDB Coordinator daemon.
 * `tasks` subsection allows to actually schedule backups (the schedule is specified in crontab format).
 
-The options within these three subsections are explained in the [Operator Options](https://percona-lab.github.io/percona-xtradb-cluster-operator/configure/operator).
+Here is an example which uses Amazon S3 storage for backups:
+
+   ```
+   ...
+   backup:
+     enabled: true
+     version: 0.3.0
+     ...
+     storages:
+       s3-us-west:
+         type: s3
+         s3:
+           bucket: S3-BACKUP-BUCKET-NAME-HERE
+           region: us-west-2
+           credentialsSecret: my-cluster-name-backup-s3
+     tasks:
+      - name: daily-s3-us-west
+        enabled: true
+        schedule: "0 0 * * *"
+        storageName: s3-us-west
+        compressionType: gzip
+     ...
+   ```
+
+   **Note:** *if you use some S3-compatible storage instead of the original Amazon S3, one more key is needed in the `s3` subsection: the `endpointUrl`, which points to the actual cloud used for backups and is specific to the cloud provider. For example, using [Google Cloud](https://cloud.google.com) involves the following one: `endpointUrl: https://storage.googleapis.com`.
+
+The options within these three subsections are further explained in the [Operator Options](https://percona-lab.github.io/percona-xtradb-cluster-operator/configure/operator).
 
 ## Making on-demand backup
 
 To make on-demand backup, user should run [the PBM Control tool](https://github.com/percona/percona-backup-mongodb#pbm-control-pbmctl) inside of the coordinator container, supplying it with needed options, like in the following example:
 
-   ```
-   kubectl run -it --rm pbmctl --image=percona/percona-server-mongodb-operator:0.2.1-backup-pbmctl --restart=Never -- --server-address=my-cluster-name-backup-coordinator:10001 run backup --destination-type=aws --compression-algorithm=gzip --description=my-backup
-   ```
+```bash
+    kubectl run -it --rm pbmctl --image=percona/percona-server-mongodb-operator:0.3.0-backup-pbmctl --restart=Never -- \
+      run backup \
+      --server-address=<cluster-name>-backup-coordinator:10001 \
+      --storage $storage \
+      --compression-algorithm=gzip \
+      --description=my-backup```
+```
```

Comment:
it is needed to add explanation about `storage` option and about `server-address`